scripts/stac/imagery/capture_area.py

```diff
@@ -10,6 +10,36 @@
 """
 
 
+def gsd_to_float(gsd: str) -> float:
```

Comment:
Could we require the parameter to be a decimal number (in metres), so that we don't have to deal with the unit? We don't support any other units, after all.

Reply:
I have to check how it is used by the other tasks (if they need the unit, like path generation in argo-task). Ideally we would like the user to pass only one parameter for the GSD.

Reply:
I have checked and it is actually not used by any other tasks (except create-collection). 
The [gsd is extracted from the TIFF](https://github.com/linz/argo-tasks/blob/0c3e4a60b7ec8044accda9050199363b876e9e0f/src/commands/path/path.generate.ts#L173) in argo-tasks, which makes me thinking that we could potentially do the same here, rather than passing it as a parameter.

Reply:
This should be dealt in another ticket/PR -> TDE-1085